pypit/data/settings/settings.kast_red

```diff
@@ -1,3 +1,32 @@
+# New Hamamatsu detector
```

Comment:
I wiped alot of lines away here.  But i did keep the 
detector name (I'll forget it).